output/mapping/eforms/mapping-tables/form-type.csv

```diff
@@ -1,7 +1,8 @@
 eForms code,eForms label,Release tag,Tender status
-planning,Procurement preparation and set-up.,'tender','planned'
+consultation,Consultation,'planning','planning'
+planning,Procurement preparation and set-up.,'tender;,'planning'
 competition,"The contracting authority or contracting entity may limit the number of candidates or tenderers to an appropriate level, on condition that this is done in a transparent manner and on the basis of objective criteria. The number of candidates or tenderers invited shall be sufficient to ensure genuine competition.",'tender','active'
-change,Replacement of a previous notice with different information.,'tenderUpdate',
```

Comment:
I'm not sure about removing the deprecated code. What if an OCDS for eForms implementation includes notices that use the deprecated code?

Reply:
we've been removing deprecated codes in other tables so removing this one would be consistent, e.g. https://github.com/open-contracting/european-union-support/commit/13d9f12969fb7d866a62f7908e4506a0e7fe18de#diff-1a45516eadbe4b0622bb92ce1d65a00ee117f6af1019b663513d001f951f7669

Reply:
I think OK to remove. Maintaining mappings for multiple versions of eForms is too high effort.